backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/calendar/internal/BpkCalendarDayCell.kt

```diff
@@ -82,6 +84,7 @@ internal fun BpkCalendarDayCell(
                 interactionSource = remember { MutableInteractionSource() },
             )
             .semantics {
+                text = AnnotatedString(if (inactive) "inactive" else "active")
```

Comment:
what happens with screenreaders here? does it announce this?

Reply:
I have double check it and screenreader doesn't announce it

Reply:
Hmm, I'm seeing/hearing it announced
![Screenshot_20241129_091246](https://github.com/user-attachments/assets/d8bd7013-43af-4a1e-a337-81d745617f67)
![Screenshot_20241129_091224](https://github.com/user-attachments/assets/bfde5ab5-ea51-45fd-b769-59a3ab1c5939)

Reply:
ok we definitely can't do this then